api/src/main/java/org/eclipse/microprofile/rest/client/spi/package-info.java

```diff
@@ -23,5 +23,5 @@
  * provide additional functionality for MP Rest Clients.
  *
  */
-@org.osgi.annotation.versioning.Version("1.1")
```

Comment:
any reason why a patch over minor?

Reply:
I was thinking about removing the `setInstance` method per issue #103 - which would force the package version to be 2.0, but then I realized that we hadn't actually come to a consensus on that.  Since this change isn't changing any method signatures, it shouldn't require a minor update. I'm not an expert in semantic versioning, but I think in this case, a micro version update is correct. 

I'd also like to separate the "spi" package into it's own artifact - that might require a version change too, but I'm not positive about that.

Reply:
Took a quick look at semVer and it probably does align with a patch version